### ISSUE TITLE
DocExpansion Enumeration

### ISSUE THREAD

**Issue by moraneva** [domaindrivendev/Swashbuckle.AspNetCore#513]
When configuring the Swagger UI options, the `DocExpansion` function should take in an enumeration instead of a string. 

Current State:
```
app.UseSwaggerUI(c =>
{
      c.DocExpansion("full");
});
```

Desired State:
```
app.UseSwaggerUI(c =>
{
      c.DocExpansion(DocExpansion.Full);
});
```

This prevents user error and provides clarity when trying to set up the Expansion type.

**Comment by moraneva** [comment]
PR for this issue is here: https://github.com/domaindrivendev/Swashbuckle.AspNetCore/pull/514

**Comment by domaindrivendev** [comment]
This feature was introduced with the updates to support swagger-ui 3x and is available in the latest version (2.3.0)
